python/pyarrow/parquet.py

```diff
@@ -1287,7 +1296,9 @@ def write_table(table, where, row_group_size=None, version='1.0',
                 coerce_timestamps=None,
                 allow_truncated_timestamps=False,
                 data_page_size=None, flavor=None,
-                filesystem=None, **kwargs):
+                filesystem=None,
+                compression_level=np.iinfo(np.intc).min,
```

Comment:
Hm, I'm wondering if we should switch compression.h to use INT32_MIN for the sentinel value rather than a platform integer. @pitrou?

Reply:
Ah, this actually probably be None instead.